acos_client/v30/slb/template/ssl.py

```diff
@@ -52,8 +52,8 @@ def _set(self, name, cert="", key="", passphrase="", update=False,
         else:
             obj_params = {
                 "name": name,
-                "cert": cert,
-                "key": key,
+                "cert-str": cert,
+                "key-str": key,
```

Comment:
Pretty sure there is a history issue going on. Let's remove this change. This change was already made in this branch: https://github.com/a10networks/acos-client/blob/stable/acos_4_1_4/acos_client/v30/slb/template/ssl.py#L56